src/generateRandomNumber.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const generateRandomNumber = () => {
+  const randomDigits = [];
+
+  randomDigits.push(Math.floor(Math.random() * (5 - 3 + 1)) + 3);
+  randomDigits.push(Math.floor(Math.random() * (7 - 6 + 1)) + 6);
+  randomDigits.push(Math.floor(Math.random() * (9 - 8 + 1)) + 8);
+  randomDigits.push(Math.floor(Math.random() * (2 - 0 + 1)) + 0);
```

Comment:
you can optimize this and create 4 digits number instant using Math.random (const randomNumber = Math.floor(Math.random() * 9000) + 1000;)

Reply:
now your can not have every number, consider implement my suggestion code

Reply:
not fixed

Reply:
i don`t know exacly what you mean

is my latest fix right?